templates/hpc/_hpc-contact-us-form.html

```diff
@@ -0,0 +1,121 @@
+<section class="p-strip">
+    <div class="u-fixed-width">
+      <h1>{{h1}}</h1>
+      <p class="p-heading--4">{{intro_text}}</p>
+    </div>
+    <div class="row">
+      <div class="col-8">
+        <form action="/marketo/submit" method="post" id="mktoForm_{{formid}}">
+          <fieldset class="u-no-margin--bottom">
+            <h3>About you</h3>
+            <ul class="p-list">
+              <li class="p-list__item">
+                <label for="firstName">First name:</label>
+                <input required id="firstName" name="firstName" maxlength="255" type="text"/>
+              </li>
+              <li class="p-list__item">
+                <label for="lastName">Last name:</label>
+                <input required id="lastName" name="lastName" maxlength="255" type="text"/>
+              </li>
+              <li class="p-list__item">
+                <label for="email">Email address:</label>
+                <input required id="email" name="email" maxlength="255" type="email" pattern="^[^ ]+@[^ ]+\.[a-z]{2,26}$" />
+              </li>
+              <li class="p-list__item">
+                <label for="phone">Phone number:</label>
+                <input required id="phone" name="phone" maxlength="255" type="tel"/>
+              </li>
+              <!-- country -->
+              {% include "shared/forms/_country.html" %}
+            </ul>
+          </fieldset>
+          <fieldset class="u-no-margin--bottom">
+            <h3>About your role</h3>
+            <ul class="p-list">
+              <li class="p-list__item">
+                <label for="jobTitle">Job title:</label>
+                <input required id="jobTitle" name="title" maxlength="255" type="text" />
+              </li>
+            </ul>
+          </fieldset>
+          <fieldset class="u-no-margin--bottom">
+            <h3>About your company</h3>
+            <ul class="p-list">
+              <li class="p-list__item">
+                <label for="company">Company name:</label>
+                <input required id="company" name="company" maxlength="255" type="text" />
+              </li>
+              <li class="p-list__item">
+                <label for="industry">Industry:</label>
+                <input required id="industry" name="title" maxlength="255" type="text" />
```

Comment:
According to copy doc, industry is not required
```suggestion
                <input id="industry" name="title" maxlength="255" type="text" />
```